frontend/src/utils/api/user.js

```diff
@@ -14,8 +14,12 @@ export function userMapper({
   joined,
   ...rest
 }) {
+  if (!joined) {
+    return
```

Comment:
return nothing?

Reply:
shouldn't it return `rest` or `{ ...rest }` ?

Reply:
just to stop errors happening idk

Reply:
Hmm. Yeah I dunno. To me it implies the user doesn't exist, and other fields will also error.

Reply:
`joined` is compulsory I'm pretty sure.